<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TileShapeGroupDialog</class>
 <widget class="QDialog" name="TileShapeGroupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tile Shape Groups</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="groupsList">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>120</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="groupsToolBarLayout"/>
     </item>
     <item>
      <widget class="Tiled::Internal::VirtualTilesetView" name="shapesList"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="shapeToolBarLayout"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Group:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="groupCombo"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Tiled::Internal::VirtualTilesetView" name="shapeSrcList"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="shapeSrcToolBarLayout"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAddGroup">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/document-new.png</normaloff>:/images/16x16/document-new.png</iconset>
   </property>
   <property name="text">
    <string>Add Group</string>
   </property>
  </action>
  <action name="actionRemove_Group">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/edit-delete.png</normaloff>:/images/16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove Group</string>
   </property>
  </action>
  <action name="actionClearShape">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/edit-clear.png</normaloff>:/images/16x16/edit-clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear Shape</string>
   </property>
   <property name="toolTip">
    <string>Remove selected shapes from current group</string>
   </property>
  </action>
  <action name="actionAddShape">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/document-new.png</normaloff>:/images/16x16/document-new.png</iconset>
   </property>
   <property name="text">
    <string>Add Shape</string>
   </property>
  </action>
  <action name="actionEditGroup">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/document-properties.png</normaloff>:/images/16x16/document-properties.png</iconset>
   </property>
   <property name="text">
    <string>Edit Group</string>
   </property>
  </action>
  <action name="actionRemoveShape">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/edit-delete.png</normaloff>:/images/16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove Shape</string>
   </property>
   <property name="toolTip">
    <string>Destroy selected shape</string>
   </property>
  </action>
  <action name="actionShapeProperties">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/document-properties.png</normaloff>:/images/16x16/document-properties.png</iconset>
   </property>
   <property name="text">
    <string>Shape Properties</string>
   </property>
  </action>
  <action name="actionMoveGroupUp">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/go-up.png</normaloff>:/images/16x16/go-up.png</iconset>
   </property>
   <property name="text">
    <string>Move Up</string>
   </property>
   <property name="toolTip">
    <string>Move the selected group up the list</string>
   </property>
  </action>
  <action name="actionMoveGroupDown">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/go-down.png</normaloff>:/images/16x16/go-down.png</iconset>
   </property>
   <property name="text">
    <string>Move Down</string>
   </property>
   <property name="toolTip">
    <string>Move the selected group down the list</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tiled::Internal::VirtualTilesetView</class>
   <extends>QTableView</extends>
   <header>virtualtilesetview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="tiled.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>TileShapeGroupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
